src/main/resources/interface-specification.yml

```diff
@@ -858,6 +860,11 @@ components:
           type: string
         path:
           type: string
+        checksum:
+          type: string
+        uploadDate:
+          type: string
+          format: date-time
```

Comment:
this makes easier testing the read model, but anyway it seems awkward to have a management service that does not expose data

Reply:
Ok sounds reasonable.